packages/dev/s2-docs/pages/s2/styling.mdx

```diff
@@ -362,4 +333,22 @@ CSS resets are strongly discouraged. Global CSS selectors can unintentionally af
 /* App.css */
 @layer reset, _;
 @import "reset.css" layer(reset);
-```
\ No newline at end of file
+```
+
+## Developing with style macros
+
+Since `style` macros are quite different from using `className`/`style` directly, many may find it initially challenging to debug and develop against.
+Below are some useful tools that may benefit your developer experience:
```

Comment:
I don't think we need to say this. We don't want to scare people.

"These tools improve the developer experience when using style macros:"